### PR TITLE
LRQA-73479 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-73479

### CODE REVIEW

**Comment by melodywu** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/macros/Organization.macro [hunk 1]
```diff
@@ -240,10 +234,6 @@ definition {
 				value1 = "${userGender}");
 		}
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/macros/Organization.macro:236]: Hi @melodywu, any reason you did not want to include these in your new macro?

Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/macros/Organization.macro:236]: Yes， you are right. At first I think I should add “Mandatory filed” but I am wrong, If I add if condition sentence I can add the other field.

**Comment by melodywu** [comment]
ci:test:sf

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]
Hi @melodywu! Looks good, I just left a few comments. Please validate and SF before re-sending. We can also run the test suite for all Users and Orgs tests given that the macros we are updating are widely used. Thanks!

File: portal-web/test/functional/com/liferay/portalweb/macros/User.macro [hunk 2]
```diff
@@ -255,61 +328,17 @@ definition {
```
[the remark this answers is not in the document]
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/macros/User.macro:328]: Ok thanks

File: portal-web/test/functional/com/liferay/portalweb/macros/User.macro [hunk 3]
```diff
@@ -354,45 +383,21 @@ definition {
...
-
-			if (isSet(userComment)) {
-				AssertTextEquals(
-					locator1 = "TextArea#COMMENTS",
-					value1 = "${userComment}");
+			User._editUserInformation(
+				userEmailAddress = "${userEmailAddress}",
+				userFirstName = "${userFirstName}",
+				userLastName = "${userLastName}",
+				userScreenName = "${userScreenName}");
```
[the remark this answers is not in the document]
Comment (melodywu) [reply portal-web/test/functional/com/liferay/portalweb/macros/User.macro:390]: Ok thanks

**Pull request closed without merging**